carbonserver/carbonserver/api/infra/database/database_manager.py

```diff
@@ -37,23 +35,28 @@ def session(self) -> Callable[..., AbstractContextManager]:
 
         except exc.IntegrityError as e:
             session.rollback()
+            logger.error(e.orig.args[0], exc_info=True)
             raise DBException(
-                error=DBError(code=DBErrorEnum.INTEGRITY_ERROR, message=e.orig.args[0])
+                error=DBError(
+                    code=DBErrorEnum.INTEGRITY_ERROR, message="Relation not found"
```

Comment:
How can I test this? Is there a particular scenario where this exception is thrown?

Reply:
Trying to create a run with an inexistant project id should throw this